php/kucoin2.php

```diff
@@ -492,6 +492,9 @@ public function fetch_deposit_address ($code, $params = array ()) {
         $currencyId = $this->currencyId ($code);
         $request = array ( 'currency' => $currencyId );
         $response = $this->privateGetDepositAddresses (array_merge ($request, $params));
+        if ($response['code'] == 200000 && !isset($response['data'])) {
```

Comment:
Hi! Thanks for your involvement! One question, though ) Can you please move this edit to the `.js` file?

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

Reply:
Also, the proper place for it is `handleErrors`, not in the `fetchDepositAddress` itself...